core/src/main/java/org/apache/calcite/avatica/util/AbstractCursor.java

```diff
@@ -1241,11 +1280,13 @@ static class TimestampAccessor extends ObjectAccessor {
    */
   static class TimestampFromUtilDateAccessor extends ObjectAccessor {
     private final Calendar localCalendar;
+    private final boolean fixedInstant;
 
-    TimestampFromUtilDateAccessor(Getter getter,
-        Calendar localCalendar) {
+    TimestampFromUtilDateAccessor(
+        Getter getter, Calendar localCalendar, boolean fixedInstant) {
       super(getter);
```

Comment:
This seems impossible. How could a `java.util.Date` represent anything but an instant?